### DISCUSSION TITLE
Please ensure that you are using the latest version of the official game releases.

### DISCUSSION THREAD

**Discussion by SylvyrrocK** [ppy/osu#30434] [Q&A] [answered]
After fresh installing osu lazer all the time i am getting this: 
"Please ensure that you are using the latest version of the official game release
Play in this state is not permitted"
Scores are not updating, multiplayer lobbies are not available, same for playlists
Currently have 2024.312.1-lazer which is latest version as for now. 
Already tried to reinstall osu, turning off windows defender, deleting some files which was suggested on other forum pages but nothing seems to help.
Any ideas on how to fix it, stuck with this problems for several days as for now.
https://gyazo.com/c8c6955331a14895cc4bfce19e9bc984
![c8c6955331a14895cc4bfce19e9bc984](https://github.com/user-attachments/assets/34915c21-9b1d-4e1b-94da-6dfd5baa4e06)

**Comment by BlockHead007** [comment, marked as answer]
You are months away from latest version which is 2024.1009.1 for now. Please update the game.

**Comment by SylvyrrocK** [comment]
Super weird, cause pressing in game check for update gives me prompt that i am running latest version.
Anyway fresh install helped this time, thanks.
